### DISCUSSION TITLE
Rocky Linux Support

### DISCUSSION THREAD

**Discussion by immauss** [immauss/openvas#254] [General]
I've recently come to find out that Greenbone has no plan of supporting Rocky in the Community feed. If you are a user of Rocky, or even if you aren't but you are using this container, I would encourage you to visit the GB forum, and upvote / pile on to this thread. Their logic on this issue is severely flawed. 

https://forum.greenbone.net/t/rocky-linux-support/16872

Thanks,
Scott

**Comment by kjake** [comment]
Agree that's a super weird decision.  I view Rocky as the spiritual successor to CentOS.

**Comment by immauss** [comment]
Right!?!?! 

Make sure you pop into the forum and make a comment if you have the time. 

Thanks,
Scott

**Reply by kjake** [reply]
You know what's wild?  I registered an account on their forum and made a post in reply to your thread... went back today to see if there's been any update, and my account and post no longer exists...

**Reply by immauss** [reply]
That is odd ....

But .... hopefully, with in the next week or so, I'll have a solution for this issue. 
:D
